### ISSUE TITLE
Plugins errors after gulp build process

### ISSUE THREAD

**Issue by kovart** [hakimel/reveal.js#2985]
I'm getting plugins errors that fail running my presentation.
```
notes.js:1 Uncaught TypeError: R is not a function
    at notes.js:1
    at notes.js:1
    at notes.js:1
(anonymous) @ notes.js:1
(anonymous) @ notes.js:1
(anonymous) @ notes.js:1
markdown.js:1 Uncaught TypeError: I is not a function
    at markdown.js:1
    at markdown.js:1
    at markdown.js:1
(anonymous) @ markdown.js:1
(anonymous) @ markdown.js:1
(anonymous) @ markdown.js:1
highlight.js:1 Uncaught TypeError: qs is not a function
    at highlight.js:1
    at highlight.js:1
    at highlight.js:1
(anonymous) @ highlight.js:1
(anonymous) @ highlight.js:1
(anonymous) @ highlight.js:1
(index):36 Uncaught ReferenceError: RevealMarkdown is not defined
    at (index):36
```
Reveal.js 7.1.2
System: Windows 10 + node v14.15.4 + npm 7.20.0

Steps to repdroduce:
Download 7.1.2 zip archive (or master)
`npm i`, `npm run build`, `npm run start`
Open localhost:8000 and see the errors in a browser console.

Replacing the plugins from the archive resolves the issue.

**Comment by souchi** [comment]
I have same issue on 4.1.2 version.

- Reveal.js:4.1.2
- System:
  OS: WIndows 10 10.0.18362
- Binaries:
  Node: 14.15.4
  npm: 6.14.13
- Browser:
  Firefox 90.0(64bit)

**Comment by vanch3d** [comment]
Having the same problem 
- Reveal 4.1.3
- Windows 10
- Node v14.17.3
- npm 7.14.0

**Comment by eleloi** [comment]
I'm getting the same problem. Without a build process it works, as vanch3d says, but I cannot build a release to serve on a web server without building it. Isn't it so?

**Comment by ManticSic** [comment]
We have the same issue. Any updates on this topic?

**Comment by souchi** [comment]
I changed the browserlist settings and it worked.

`
"browserslist": "> 2%, not dead",
`

**Comment by hakimel** [comment]
Thanks @souchi. No point in supporting IE11 anymore so I've update the browserlist in master. Has the added benefit of reducing the build times by 50%.

Can someone else who has encountered this issue test with the new browserlist to confirm that it works?

**Comment by darekkay** [comment]
> Can someone else who has encountered this issue test with the new browserlist to confirm that it works?

I can confirm this. I've reproduced the issue on 039972c and it's fixed on b23d15c 🚀

**Comment by hakimel** [comment]
Thanks @darekkay!
